flake-modules/dev/default.nix

```diff
@@ -52,6 +52,9 @@
             "**.scm"
             "**.svg"
             "**/man/*.5"
+            # Those files are generated by pytest-regression, which then `diff`s them.
+            # Formatting them will make the tests fail.
+            "/docs/gfm-alerts-to-admonitions/tests/**/*.yml"
```

Comment:
Seems to not be working. Maybe the leading `/` is incorrect?

https://buildbot.nix-community.org/#/builders/720/builds/1095/steps/1/logs/stdio